.github/workflows/publish_docs.yml

```diff
@@ -22,6 +22,7 @@ jobs:
             - name: Install doc dependencies
               run: |
                   pip install -r docs/requirements.txt
+                  pip install .
```

Comment:
not so sure if this is best practice. Shouldn't we add missing deps to docs/requirements.txt instead?

Reply:
Feel free to merge if this satisfies!